rest_framework/utils/mediatypes.py

```diff
@@ -85,5 +84,5 @@ def precedence(self):
     def __str__(self):
         ret = "%s/%s" % (self.main_type, self.sub_type)
         for key, val in self.params.items():
-            ret += "; %s=%s" % (key, val)
+            ret += "; %s=%s" % (key, val.decode(HTTP_HEADER_ENCODING))
```

Comment:
Not obvious to me that this behaviour needs changing - unless we've an open bug related to it?